### PR TITLE
Presenter test

### PR DESCRIPTION
**Pull request by dylan-muszel**
### Summary
- Adding WolmoPresenterTest
- Removing Activity key that was not working

### CODE REVIEW

File: core/build.gradle [hunk 1]
```diff
@@ -1,4 +1,6 @@
 apply plugin: 'com.android.library'
+apply plugin: 'kotlin-android-extensions'
```
Comment (emalamela) [inline core/build.gradle]: Why was `kotlin-android-extensions` included here?

File: core/src/main/java/ar/com/wolox/wolmo/core/tests/WolmoPresenterTest.kt [hunk 2]
```diff
@@ -0,0 +1,29 @@
+package ar.com.wolox.wolmo.core.tests
+
+import ar.com.wolox.wolmo.core.presenter.BasePresenter
+import org.junit.Before
+import org.mockito.Mockito
+import org.mockito.MockitoAnnotations
+import java.lang.reflect.ParameterizedType
+
+abstract class WolmoPresenterTest<V : Any, P : BasePresenter<V>> {
```
Comment (emalamela) [inline core/src/main/java/ar/com/wolox/wolmo/core/tests/WolmoPresenterTest.kt]: You should comment this. 😉

File: core/build.gradle (hunk 1 above)
Comment (dylan-muszel) [reply core/build.gradle]: It was auto generated when I added Kotlin. I'll remove it 😅

Comment (emalamela) [reply core/build.gradle]: LOL.

**Pull request merged**